src/workspace.js

```diff
@@ -1065,6 +1065,12 @@ module.exports = class Workspace extends Model {
         if (typeof item.setCursorBufferPosition === 'function') {
           item.setCursorBufferPosition([initialLine, initialColumn])
         }
+        if (typeof item.unfoldBufferRow === 'function') {
+          item.unfoldBufferRow(initialLine)
+        }
+        if (typeof item.scrollToBufferPosition === 'function') {
```

Comment:
This conditional and the one above are inspired by this pre-existing conditional:

https://github.com/atom/atom/blob/0c51771244ae07ab383a089591788ba2ea9726f3/src/workspace.js#L1065-L1067

That conditional ☝️ was added in 11112cb374ee0afc5f6a3a13fe3827e6e6425441 to support the scenario "where a custom editor is opened that isn't a text editor" so that "it won't blow up if the custom editor doesn't implement `setCursorBufferPosition`."

That same motivation seems relevant to the calls to `unfoldBufferRow` and `scrollToBufferPosition` introduced in this pull request, so I've followed the same pattern.